.drone.yml

```diff
@@ -85,7 +85,7 @@ steps:
       chart_version: 4.12.0
       add_repos: [mongodb=https://10gen.github.io/helm-charts]
       namespace: devrel
-      release: devcenter-frontend
+      release: devcenter-frontend-1480
```

Comment:
Doing this since the previews are not working again https://github.com/mongodb/devcenter/settings/hooks/367227055?tab=deliveries .. not sure why.